insalan/tournament/manage/match.py

```diff
@@ -0,0 +1,12 @@
+from ..models import Match, Score, MatchStatus
+
+def update_match_score(match: Match, data):
+    match.times = data["times"]
```

Comment:
all of the checking of keys and extracting of values + verification of types should happen in views as early as possible, so that the view itself can return HTTP codes and does not have to let underlying methods know what they're being called by.

i.e. here you probably should not have `data` as a single argument with the expectation that "times" or "score" are correct, but already provide them in the arguments.